numpy/__config__.pyi

```diff
@@ -81,7 +82,7 @@ _ConfigDict = TypedDict(
 
 ###
 
-__all__ = ["show"]
+__all__: Final[Sequence[str]] = ["show_config"]
```

Comment:
The annotatioon isn't needed here. `__all__` has special meaning in stubs:

https://typing.readthedocs.io/en/latest/guides/writing_stubs.html#all

I'm not sure if this matters in practice, but it save an import 🤷🏻

Reply:
Then it isn't needed - reverted!